src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts

```diff
@@ -53,21 +54,25 @@ class HttpError extends Error {
     this.statusText = statusText;
   }
 }
+export interface McFusUploader {
+  Start(file: McFusFile): void;
+  Cancel(): void;
+}
 
-export const McFusUploader: any = function (this: any, args: IInitializeSettings) {
-  let ambiguousProgress = 0;
-  let progressUpdateRate = 0;
-  const maxNumberOfConcurrentUploads = 10;
+export class McFusNodeUploader implements McFusUploader {
+  ambiguousProgress: number = 0;
+  progressUpdateRate: number = 0;
+  maxNumberOfConcurrentUploads: number = 10;
 
-  const uploadBaseUrls = {
+  uploadBaseUrls: any = {
     CancelUpload: "upload/cancel/",
     SetMetadata: "upload/set_metadata/",
     UploadChunk: "upload/upload_chunk/",
     UploadFinished: "upload/finished/",
     UploadStatus: "upload/status/",
   };
 
-  const uploadData: IUploadData = {
+  uploadData: IUploadData = {
```

Comment:
Why do you remove const here?

Reply:
There are no "const" class members in typescript. I can only make it `readonly`.